File: solutions/uri/1 - Iniciante/1010.py
class Peca:
    def __init__(self, codigo, quantidade, valorUnit):
        self.codigo = codigo
        self.quantidade = float(quantidade)
        self.valorUnit = float(valorUnit)

    def soma(self):
        return self.quantidade * self.valorUnit

entrada = input().split()
peca1 = Peca(*entrada)
entrada = input().split()
peca2 = Peca(*entrada)
print("VALOR A PAGAR: R$", format(peca1.soma() + peca2.soma(), '.2f'))

File: solutions/uri/1010/1010.rb
class Peca
	@codigo
	@quantidade
	@valorUnit
	def initialize(codigo, quantidade, valorUnit)
		@codigo = codigo
		@quantidade = quantidade
		@valorUnit = valorUnit.to_f
	end

	def print
		puts @codigo
		puts @quantidade
		puts @valorUnit
	end
	
	def soma
		@quantidade.to_f * @valorUnit
	end
end

entrada = gets.chomp.split(" ")
peca1 = Peca.new(entrada[0], entrada[1], entrada[2])

entrada = gets.chomp.split(" ")
peca2 = Peca.new(entrada[0], entrada[1], entrada[2])

puts "VALOR A PAGAR: R$ " + ('%.2f' % (peca1.soma + peca2.soma))